source/install/desktop.rst

```diff
@@ -16,6 +16,8 @@ Below is a list of additional resources:
  - `Changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_
  - Contributor’s guide (coming soon)
  - `Source code <https://github.com/mattermost/desktop>`_
+ 
+If you have any problems installing Mattermost, see the `troubleshooting guide <https://www.mattermost.org/troubleshoot/>`_. To submit an improvement or correction, click Edit at the top of this page.
```

Comment:
This should probably be "installing the desktop app" instead of installing Mattermost. Also, this page already has a troubleshooting section. You should try to integrate this note with it.